css/shopping-cart.css

```
@@ -0,0 +1,118 @@
+
+/* Movil style */
+.shopping-cart {
+  width: var(--width);
+  padding: 0px;
+}
+
+.shopping-cart__top {
+  display: flex;
+  padding-top: 15px;
+  padding-bottom: 24px;
+  gap: 9px;
+  margin: 0 16px; 
+}
+
+.shopping-cart__arrow-right {
+  width: 8px;
+  height: 8px;
+  border-right: 2.3px solid var(--color-soft-gray);
+  border-bottom: 2.3px solid var(--color-soft-gray);
+  transform: rotate(-45deg);
+  margin-top: 5px;
+}
+
+.shopping-cart__text {
+  color: var(--color-soft-gray);
+  font-size: var(--font-size-smalls);
+  font-weight: var(--font-weight);
+  line-height: var(--eighteen-pixel);
+  letter-spacing: -1%;
+}
+
+.shopping-cart__bottom__background {
+  background-color: var(--color-white);
+  height: auto; 
+}
+
+.shopping-cart__bottom__add {
+  display: flex;
+  flex-direction: column; 
+  align-items: flex-start; 
+  padding: var(--padding-vertical);
+}
+
+.shopping-cart__bottom {
+  margin: 0 16px; 
+}
+
+.shopping-cart__bottom-left {
+  display: flex;
+  flex-direction: column;
+  gap: var(--twenty-pixel);
+}
+
+.shopping-cart__left-text {
+  font-size: var(--font-size-xlarge);
+  font-weight: var(--font-weight-bold);
+  line-height: 28px;
+  letter-spacing: 1%;
```

Igual aca, se marca como error. Ademas, en el diseño esta -1%
```suggestion
  letter-spacing: -0.01em;
```